apertium-eng.eng.rlx

```diff
@@ -311,6 +312,10 @@ SELECT Num IF (1 N) ;
 SELECT Num IF (1 Adj) (2 N) ;
 SELECT Num IF (1 Adv) (2 Adj) (3 N) ;
 
+# Has (Cnjsub) and has (Det)
+SELECT Det IF (0 Cnjsub) (1 N) (2 Vbser) ;
+SELECT Det IF (0 Cnjsub) (-1 Vbser) ;
+SELECT Det IF (0 Cnjsub) (1 Vbser) ;
```

Comment:
Does this particular rule make sense?  Could it also be "...the one that is..." ?

Reply:
You could probably also select Cnjsub if there's a det after it, e.g. "...that the cat is...".

Reply:
Hm, that's true.  Maybe I'll remove this rule and replace it with the one you mentioned.